Neos.ContentRepository.Core/Classes/SharedModel/Node/NodeAggregateId.php

```diff
@@ -50,6 +50,23 @@ public static function fromString(string $value): self
         return new self($value);
     }
 
+    /**
+     * Calculates a deterministic nodeAggregateId.
+     *
```

Comment:
used for new nodes, but previously created tethered node ids were random

comment